templates/cisco_ios_show_ip_cef_detail.textfsm

```diff
@@ -0,0 +1,126 @@
+# Even though this template understands the output from
+#   - show ip cef [vrf NAME] [PREFIX | INTERFACE] [detail]
+# INDEX file forces you to use 'detail' keyword, or there
+# will be no match. This is done to simplify INDEX file
+# matching logic, otherwise matching regex is too complex.
+#
+# This template is compatible with IOS v12.x and above,
+# including IOS-XE. There are two versions of output
+# with significant differences depending on IOS version.
+# Template is able to recognize both versions and captures
+# same structured data regardless of IOS version.
+#
+Value Required PREFIX ((?:\d{1,3}\.){3}\d{1,3}\/\d{1,2})
+Value List FLAGS ([^,\]\n]+)
+Value TYPE ((?:no\s)?(?:[a-z]+))
```

Comment:
This is the same thing in the above template as well.

Separating these out is more of a post-processing step and addresses your specific use case, but with maintaining a repository for most people, we need to solve the parsing problem and not the parsing/processing of specific use cases.

Other than those small adjustments, these looks good to merge once those are fixed and we really appreciate your effort and the level of detail you have taken!

Reply:
Again, I am not quite sure what is this suggestion about

Reply:
I disagree as to this template. In detail view type is presented as SEPARATE value, it's not using same placement as NEXTHOP

Reply:
Not going to resolve this, as it's different scenario, such as
```
  ^\s+${TYPE}(?<=receive)\s+for\s+${INTERFACE}
  ^\s+${TYPE}(?<=attached)\s+to\s+${INTERFACE}
  ^\s+${TYPE}(?<=drop)$$
  ^\s+${TYPE}(?<=no route)$$
  ^\s+${TYPE}(?<=receive)$$
  ^\s+${TYPE}(?<=multicast)$$
  ^\s+${TYPE}(?<=nexthop)\s+${NEXTHOP}\s+${INTERFACE}
```